app/Modules/Group/Events/Traits/IsPublishableApplicationEvent.php

```diff
@@ -18,16 +18,28 @@ public function getEventType(): string
     }
 
     public function getPublishableMessage(): array
-    {
-        return [
-            'expert_panel' => [
-                'id' => $this->group->uuid,
-                'name' => $this->group->displayName,
-                'type' => $this->group->fullType->name,
-                'affiliation_id' => $this->group->expertPanel->affiliation_id
-            ],
-        ];
-    }
+{
+    return [
+        'expert_panel' => array_merge([
+            'id' => $this->group->uuid,
+            'long_name' => $this->group->name,
+            'short_name' => $this->group->expertPanel->short_base_name,
+            'status' => optional($this->group->groupStatus)->name, // Retrieve the status name
+            'group updated date' => $this->group->updated_at,
+            'parent_group' => optional($this->group->parentGroup)->name,
+            'type' => $this->group->fullType->name,
+            'affiliation_id' => $this->group->expertPanel->affiliation_id,
+            'scope description' => $this->group->expertPanel->scope_description,
+        ],
+        // Conditionally add vcep fields below if type is 'vcep'
+        $this->group->fullType->name === 'vcep' ? ['clinvar_id' => null] : [],
+        $this->group->fullType->name === 'vcep' ? ['clinvar_url' => null] : [],
+        $this->group->fullType->name === 'vcep' ? ['cspec_url' => $this->group->expertPanel->affiliation_id] : [],
```

Comment:
This is just an affiliation id, not a URL, so the field should probably have a different name. I thought we had talked about the website team generating this url from the affiliation ID, right?